jbpm-human-task/jbpm-human-task-core/pom.xml

```diff
@@ -333,4 +328,19 @@
     </plugins>
 
   </build>
+  
+  <profiles>
+    <profile>
+      <id>jkd-11-compile</id>
+      <activation>
+        <jdk>[9,)</jdk>
```

Comment:
Actually I think it was deleted only from JDK 11 onwards. From JDK 9 onwards it was just deprecated for removal. But since we support just 8 and 11 I think it's fine.